design/ceph/object/store.md

```diff
@@ -60,6 +60,8 @@ The pools are the backing data store for the object store and are created with s
 
 If `preservePoolsOnDelete` is set to 'true' the pools used to support the object store will remain when the object store will be deleted. This is a security measure to avoid accidental loss of data. It is set to 'false' by default. If not specified is also deemed as 'false'.
 
+If user want to use existing pools for metadata and data, the pools must be created before the object store is created. This will be useful if multiple objectstore can share same pools. The detail of pools need to shared in `radosNamespaces` settings in object-store CRD. And for this option to work, `preservePoolsOnDelete` must be set to 'true'. Now the object stores can consume same pool isolated with different namespaces. User can create via [Pool CRD](/Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md), this is need to present before the object store is created.
```

Comment:
Why do the pools need to be created in advance? I would propose that there is no need for the CephBlockPool CR to be used to create the pools. Those pools are used only for rbd. If the `radosNamespaces` settings are specified in the CephObjectStore CR, then the reconcile of the object store will just create them if they don't exist yet. The first object store created would just create them. Since the controller processes the object CRs serially, there will not be a conflict. If this is the case, then we should always treat these pools as `preservePoolsOnDelete: true` since there is no object store owner for the pools.